### PR TITLE
EUDEMO reactor starting point

### PR DESCRIPTION
**Pull request by hsaunders1904**
## Linked Issues

Closes #{ID}

## Description

Some working builds

## Checklist

I confirm that I have completed the following checks:

- [ ] Tests run locally and pass `pytest tests --reactor`
- [ ] Code quality checks run locally and pass `flake8` and `black .`
- [ ] Documentation built locally and checked `sphinx-build -W documentation/source documentation/build`

### CODE REVIEW

**Review by CoronelBuendia — APPROVED** [review]

**Review by je-cook — CHANGES_REQUESTED** [review]
Looks great! Just a few things that probably didnt come over in a rebase I think

File: examples/design/EU-DEMO/EUDEMO_builders/power_cycle.py [hunk 1]
```diff
@@ -1,175 +0,0 @@
-# bluemira is an integrated inter-disciplinary design tool for future fusion
```
[the remark this answers is not in the document]
Comment (je-cook) [reply examples/design/EU-DEMO/EUDEMO_builders/power_cycle.py:1]: I think the whole design folder is gone and the rest of the files have just been moved: https://github.com/Fusion-Power-Plant-Framework/bluemira/tree/635c369845233f5af35dab739b0964310f3c5a3f/examples

**Review by je-cook — APPROVED** [review]
Happy once the tests run

**Comment by hsaunders1904** [comment]
OK, I've got nothing more to add to this, I don't think. Tests aren't expected to pass at this point, but they're now running.

**Pull request merged**